modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/SmoothProgressBar.scala

```diff
@@ -25,14 +25,23 @@ abstract class SmoothProgressBarProps[A](override val component: Scala.Component
 
 trait SmoothProgressBar[P <: SmoothProgressBarProps[P]] {
   @Lenses
-  protected case class State(maxValue: Int, value: Int, skipStep: Boolean)
+  protected case class State(value: Int, prevStopping: Boolean, prevPaused: Boolean)
```

Comment:
Was skipStep unused?

Reply:
It was being used. If I understand correctly, it was used to skip 1 50ms tick if the inferred progress was ahead.

Now that we don't have an instance where we can react to the moment properties change, it's a bit tricky to implement the logic, I'm not sure if it's possible. The drawback is that the progress *may* be (bot not necessarily) up to 1 second ahead for every 20 seconds elapsed. I don't know how admissible this is.